docs/man/docker.1.md

```diff
@@ -81,7 +81,7 @@ unix://[/path/to/socket] to use.
   Default IP address to use when binding container ports. Default is `0.0.0.0`.
 
 **--ip-forward**=*true*|*false*
-  Docker will enable IP forwarding. Default is true. If `--fixed-cidr-v6` is set. IPv6 forwarding will be activated, too. This may reject Router Advertisements and interfere with the host's existing IPv6 configuration. For more information please consult the documentation about "Advanced Networking - IPv6".
+  Docker will enable IP forwarding. Default is true. If it's set as false, Docker will leave `ip-forward` as it was, not disable it. `--fixed-cidr-v6` is set. IPv6 forwarding will be activated, too. This may reject Router Advertisements and interfere with the host's existing IPv6 configuration. For more information please consult the documentation about "Advanced Networking - IPv6".
```

Comment:
This reads better if you don't try to restrict the number of sentences.

---> https://gist.github.com/moxiegirl/dcc98bd5f02e61af8168

Enables IP forwarding on the Docker host. The default is `true` or `enabled`. This flag interacts with the IP forwarding setting on your host system's kernel. If your system has IP forwarding disabled, this setting enables it. If your system has IP forwarding disabled, setting this flag to `ip-forward=false` has no effect.  

This setting does not enable or disable IPv6 forwarding it. If `ip-forward=true` and your system's kernel enables IPv6 forwarding, you may receive reject Router Advertisements. Additionally, this setting can interfere with the host's existing IPv6 configuration. For more information, please consult the documentation about "Advanced Networking - IPv6".